.erasim/themes/solarized.theme/theme.sass

```diff
@@ -0,0 +1,47 @@
+$primary-color-dark:   #1976D2
+$primary-color:        #2196F3
+$primary-color-light:  #BBDEFB
+$primary-color-text:   #FFFFFF
+$accent-color:         #FF4081
+$primary-text-color:   #212121
+$secondary-text-color: #757575
+$divider-color:        #BDBDBD
+
+$section-height: 50px
+$section-width: 280px
+$button-height: 40px
+
+create-project
+  width: $section-width
+  margin-top: 10px
+  section
+    margin: 20px
+    height: $section-height
+    font-size: 20px
+    border-width: 1px
+    border-color: $divider-color
+  name
+    length: 80px
+  button
+    margin-top: 10px
+    background-color: $primary-color
+    width: 160px
+
+button
+  color: $primary-color-text
+  font-size: 16px
+  height: $button-height
+  radius: 4
+  border
+    color: lighten($primary-color, 20%)
+    width: 2px
+
+window
+  width: 1124px
+  height: 700px
+  background-color: white
+  initial-width: $section-width + 20px
+  initial-height: $section-height * 6 + $button-height + 10
```

Comment:
when styling the tabbar, its height should be added here as well to adjust the initial height properly on all systems.